resync/api/agent_evolution_api.py

```
@@ -205,10 +213,10 @@ async def get_patterns(agent_name: str):
         patterns = []
 
         if pattern_dir.exists():
```

Ruff (0.15.0)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)